src/navigation/MainNavigator.tsx

```diff
@@ -14,6 +15,11 @@ export const MainNavigator = () => {
           component={Connections}
           options={{headerShown: false}}
         />
+        <Stack.Screen
+          name={'BookingScreen'}
```

Comment:
By creating an enum for the screen names, you can avoid typos. Afterall this is a small project, so it's totally okay the way it is, but at a more complex project I suggest using enum.